### PR TITLE
updated dependencies so it can install

### PR DESCRIPTION
**Pull request by Sophrinix**
bumped the version number to 0.2.10 and updated the versions on the dependencies so this can install.

This impacts, amongst other programs tio2.
see this:
https://gist.github.com/Sophrinix/d7ecf08626d058201e5d

Please merge this. I've signed the CLA as y'all are aware of :-)

### CODE REVIEW

**Comment by Sophrinix** [comment]
This auto hasn't signed the CLA is wrong.

**Comment by cb1kenobi** [comment]
@Sophrinix Yeah, this duplicates #14. We are trying to get #14 done and published. Need just a bit more time.

**Comment by Sophrinix** [comment]
Is there anyway to just go ahead and push an update to unbreak tio2? That's my real interest.

**Comment by cb1kenobi** [comment]
Well, you could just change your package.json and point `ioslib` to `git+https://git@github.com/appcelerator/ioslib.git#timob-18867` and help test #14. That would help immensely.

**Comment by Sophrinix** [comment]
I did that and ran the example test in tio2. 
Are there specific tests that I should try?
This was the output:
➜  Desktop  cd tio2
➜  tio2 git:(master) ✗ tio2 ./example
2015-06-16T21:24:33.532Z - tio2:        Summary:
2015-06-16T21:24:33.535Z - tio2:        ┌───────┬────────┬────────┐
2015-06-16T21:24:33.536Z - tio2:        │ Suite │ Passed │ Failed │
2015-06-16T21:24:33.536Z - tio2:        └───────┴────────┴────────┘
2015-06-16T21:24:33.536Z - tio2:
2015-06-16T21:24:33.536Z - tio2-error:  Missing --platform <name>
2015-06-16T21:24:33.537Z - tio2:        Finished in 6ms
➜  tio2 git:(master) ✗ tio2 ./example  --platform=ios
2015-06-16T21:24:41.070Z - tio2:        Found 1 test suite:
2015-06-16T21:24:41.073Z - tio2:          foo
2015-06-16T21:24:41.074Z - tio2:
2015-06-16T21:24:41.074Z - tio2:        Running suite 1 of 1: foo
2015-06-16T21:24:41.076Z - tio2:        Build directory: /var/folders/30/l336z8_x0_d0qlth0p1ngqs80000gn/T/tio2_115516-56157-1w47vqx
2015-06-16T21:24:41.076Z - tio2:        Copying harness app files from /Users/andrewmcelroy/Desktop/tio2/example/harness...
2015-06-16T21:24:41.192Z - tio2:        Copying suite resource files from /Users/andrewmcelroy/Desktop/tio2/example/suites/foo...
2015-06-16T21:24:41.194Z - tio2:        Copying test dependency files from /Users/andrewmcelroy/Desktop/tio2/node_modules/ti-mocha...
2015-06-16T21:24:41.199Z - tio2:        Preparing tiapp.xml
2015-06-16T21:24:41.207Z - tio2:        Unzipping: /Users/andrewmcelroy/Desktop/tio2/modules/ios/com.appcelerator.timer-iphone-1.0.zip => /var/folders/30/l336z8_x0_d0qlth0p1ngqs80000gn/T/tio2_115516-56157-1w47vqx
2015-06-16T21:24:41.300Z - tio2:        Writing new app.js: /var/folders/30/l336z8_x0_d0qlth0p1ngqs80000gn/T/tio2_115516-56157-1w47vqx/Resources/app.js
2015-06-16T21:24:41.301Z - tio2:        Executing: titanium "--no-colors" "--no-progress-bars" "--no-banner" "--no-prompt" "--log-level" "trace" "--sdk" "3.5.1.GA" "build" "--project-dir" "/var/folders/30/l336z8_x0_d0qlth0p1ngqs80000gn/T/tio2_115516-56157-1w47vqx" "--build-only" "--platform" "ios"
2015-06-16T21:25:01.117Z - tio2:        Running app
2015-06-16T21:25:01.117Z - tio2:        Launching iOS app on simulator
2015-06-16T21:25:11.208Z - tio2:        Processing results
2015-06-16T21:25:11.235Z - tio2:        Cleaning up
2015-06-16T21:25:11.461Z - tio2:        Suite foo finished successfully  6 passed  0 failed
2015-06-16T21:25:11.461Z - tio2:
2015-06-16T21:25:11.461Z - tio2:        Summary:
2015-06-16T21:25:11.463Z - tio2:        ┌───────┬────────┬────────┐
2015-06-16T21:25:11.463Z - tio2:        │ Suite │ Passed │ Failed │
2015-06-16T21:25:11.463Z - tio2:        ├───────┼────────┼────────┤
2015-06-16T21:25:11.464Z - tio2:        │ foo   │ 6      │ 0      │
2015-06-16T21:25:11.464Z - tio2:        └───────┴────────┴────────┘
2015-06-16T21:25:11.464Z - tio2:
2015-06-16T21:25:11.464Z - tio2-result: {
2015-06-16T21:25:11.464Z - tio2-result: 	"summary": {
2015-06-16T21:25:11.465Z - tio2-result: 		"passed": 6,
2015-06-16T21:25:11.465Z - tio2-result: 		"failed": 0
2015-06-16T21:25:11.465Z - tio2-result: 	},
2015-06-16T21:25:11.465Z - tio2-result: 	"suites": {
2015-06-16T21:25:11.465Z - tio2-result: 		"foo": {
2015-06-16T21:25:11.465Z - tio2-result: 			"date": "2015-06-16T21:25:10.966Z",
2015-06-16T21:25:11.465Z - tio2-result: 			"results": [
2015-06-16T21:25:11.465Z - tio2-result: 				{
2015-06-16T21:25:11.465Z - tio2-result: 					"state": "passed",
2015-06-16T21:25:11.466Z - tio2-result: 					"duration": 2491,
2015-06-16T21:25:11.466Z - tio2-result: 					"suite": "test",
2015-06-16T21:25:11.466Z - tio2-result: 					"title": "should test empty view"
2015-06-16T21:25:11.466Z - tio2-result: 				},
2015-06-16T21:25:11.466Z - tio2-result: 				{
2015-06-16T21:25:11.466Z - tio2-result: 					"state": "passed",
2015-06-16T21:25:11.466Z - tio2-result: 					"duration": 1326,
2015-06-16T21:25:11.466Z - tio2-result: 					"suite": "test",
2015-06-16T21:25:11.466Z - tio2-result: 					"title": "should test view with parameters"
2015-06-16T21:25:11.467Z - tio2-result: 				},
2015-06-16T21:25:11.467Z - tio2-result: 				{
2015-06-16T21:25:11.467Z - tio2-result: 					"state": "passed",
2015-06-16T21:25:11.467Z - tio2-result: 					"duration": 487,
2015-06-16T21:25:11.467Z - tio2-result: 					"suite": "test",
2015-06-16T21:25:11.467Z - tio2-result: 					"title": "should test view with custom parameters"
2015-06-16T21:25:11.467Z - tio2-result: 				},
2015-06-16T21:25:11.468Z - tio2-result: 				{
2015-06-16T21:25:11.468Z - tio2-result: 					"state": "passed",
2015-06-16T21:25:11.468Z - tio2-result: 					"duration": 6387,
2015-06-16T21:25:11.468Z - tio2-result: 					"suite": "test",
2015-06-16T21:25:11.468Z - tio2-result: 					"title": "should test window"
2015-06-16T21:25:11.468Z - tio2-result: 				},
2015-06-16T21:25:11.468Z - tio2-result: 				{
2015-06-16T21:25:11.468Z - tio2-result: 					"state": "passed",
2015-06-16T21:25:11.468Z - tio2-result: 					"duration": 1468,
2015-06-16T21:25:11.468Z - tio2-result: 					"suite": "test",
2015-06-16T21:25:11.468Z - tio2-result: 					"title": "should test postlayout event"
2015-06-16T21:25:11.469Z - tio2-result: 				},
2015-06-16T21:25:11.469Z - tio2-result: 				{
2015-06-16T21:25:11.469Z - tio2-result: 					"state": "passed",
2015-06-16T21:25:11.469Z - tio2-result: 					"duration": 1439,
2015-06-16T21:25:11.469Z - tio2-result: 					"suite": "test",
2015-06-16T21:25:11.469Z - tio2-result: 					"title": "should be able to read foo.txt"
2015-06-16T21:25:11.469Z - tio2-result: 				}
2015-06-16T21:25:11.469Z - tio2-result: 			],
2015-06-16T21:25:11.469Z - tio2-result: 			"platform": {
2015-06-16T21:25:11.469Z - tio2-result: 				"ostype": "32bit",
2015-06-16T21:25:11.469Z - tio2-result: 				"name": "iPhone OS",
2015-06-16T21:25:11.470Z - tio2-result: 				"osname": "iphone",
2015-06-16T21:25:11.470Z - tio2-result: 				"version": "8.3",
2015-06-16T21:25:11.470Z - tio2-result: 				"address": "192.168.0.106",
2015-06-16T21:25:11.470Z - tio2-result: 				"macaddress": "4B141939-823F-4757-BE1F-4553E4567416",
2015-06-16T21:25:11.470Z - tio2-result: 				"architecture": "i386",
2015-06-16T21:25:11.470Z - tio2-result: 				"availableMemory": 288.75,
2015-06-16T21:25:11.470Z - tio2-result: 				"manufacturer": "apple",
2015-06-16T21:25:11.470Z - tio2-result: 				"model": "Simulator"
2015-06-16T21:25:11.470Z - tio2-result: 			},
2015-06-16T21:25:11.470Z - tio2-result: 			"displayCaps": {
2015-06-16T21:25:11.471Z - tio2-result: 				"density": "high",
2015-06-16T21:25:11.471Z - tio2-result: 				"dpi": 320,
2015-06-16T21:25:11.471Z - tio2-result: 				"platformHeight": 480,
2015-06-16T21:25:11.471Z - tio2-result: 				"platformWidth": 320
2015-06-16T21:25:11.471Z - tio2-result: 			},
2015-06-16T21:25:11.471Z - tio2-result: 			"build": {
2015-06-16T21:25:11.471Z - tio2-result: 				"date": "03/05/15 10:08",
2015-06-16T21:25:11.471Z - tio2-result: 				"git": "96875c9",
2015-06-16T21:25:11.472Z - tio2-result: 				"version": "3.5.1"
2015-06-16T21:25:11.472Z - tio2-result: 			}
2015-06-16T21:25:11.472Z - tio2-result: 		}
2015-06-16T21:25:11.472Z - tio2-result: 	}
2015-06-16T21:25:11.472Z - tio2-result: }
2015-06-16T21:25:11.472Z - tio2:        Finished in 30s 405ms
➜  tio2 git:(master) ✗

everything (so far looks good). keep in mind I am running node 0.10.13
➜  tio2 git:(master) ✗ node -v
v0.10.13

Also, I did not run this on a real apple watch ( I don't have one).

**Comment by Sophrinix** [comment]
TL;DR: so far so good.

**Comment by cb1kenobi** [comment]
Node.js 0.10.13 is ancient. You should be using 0.10.38.

There were changes beyond watch support.
